node/src/service.rs

```diff
@@ -147,6 +147,12 @@ where
 	let telemetry_span = TelemetrySpan::new();
 	let _telemetry_span_entered = telemetry_span.enter();
 
+  if parachain_config.offchain_worker.enabled {
```

Comment:
indentation seems off.

Reply:
Checked locally, it's tab vs spaces issue. Will write an issue separately about this.

Reply:
#27  editorconfig support needs to be added